datasets/huishoudelijkafval/ticket/v1.0.0.json

```diff
@@ -0,0 +1,249 @@
+                "description": "De naam van de eigenaar van de container."
+            },
+            "status": {
+                "type": "string",
+                "provenance": "status_ticket",
+                "description": "De status van het ticket."
+            },
+            "naamTickettype": {
+                "type": "string",
+                "provenance": "ticket_type_name",
+                "description": "De naam van het type ticket."
+            },
+            "rangordeNummerTickettype": {
+                "type": "integer",
+                "provenance": "ticket_type_rangorde_nummer",
+                "description": "Het rangordenummer van het type ticket."
+            },
+            "datumWorkflowStart": {
+                "type": "datetime",
+                "description": "De datum waarop de workflow start."
+            },
+            "datumGeaccepteerd": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket geaccepteerd is door het onderhoudsbedrijf."
+            },
+            "datumOfferte": {
+                "type": "datetime",
+                "description": "De datum waarop de offerte is aangemaakt ."
+            },
+            "datumGepland": {
+                "type": "datetime",
+                "description": "De datum waarop de uitvoering van de werkzaamheden is ingepland."
+            },
+            "datumGereed": {
+                "type": "datetime",
+                "description": "De datum waarop de werkzaamheden uitgevoerd zijn."
+            },
+            "datumAfgerond": {
+                "type": "datetime",
+                "description": "De datum waarop de werkzaamheden als gereed aangegeven zijn door het onderhoudsbedrijf."            
+             },
+            "datumTechnischeGoedkeuring": {
+                "type": "datetime",
+                "description": "De datum waarop de technische aspecten van de werkzaamheden goed- of afgekeurd zijn." 
+            },
+            "datumFinanciëleGoedkeuring": {
+                "type": "datetime",
+                "description": "De datum waarop de financiële aspecten van de werkzaamheden goed- of afgekeurd zijn." 
+            },
+            "datumGefactureerd": {
+                "type": "datetime",
+                "provenance": "datum_gefactureerd_in_systeem",
+                "description": "De datum waarop de factuur in het systeem de status 'gefactureerd' krijgt."
+            },
+            "datumAfgewezen": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket de status 'afgewezen' krijgt."
+            },
+            "redenAfgewezen": {
+                "type": "string",
+                "provenance": "afgewezen_reden",
+                "description": "De reden waarom het ticket de status 'afgewezen' heeft gekregen."
+              },
+            "prijs": { 
+                "type": "string",
```

Comment:
I would expect `prijs` to be a `number` field?

Reply:
aangepast